NEWS.md

```diff
@@ -1,6 +1,6 @@
 ## crosstalk 1.1.1.9000
 
-
+- Remove Bootstrap dependency
```

Comment:
Should also mention the updates to selectize.js and ionRangeSlider

Reply:
Updated, thanks.